### DISCUSSION TITLE
When is https://cdn.coollabs.io/coolify/versions.json getting updated?

### DISCUSSION THREAD

**Discussion by nilsingwersen** [coollabsio/coolify#2453] [Q&A] [answered]
The domain mentioned above is responsible for updating coolify if I understand it correctly. When is it getting updated? Is there a reason for it not being updated on release of a new beta version?

**Comment by peaklabs-dev** [comment, marked as answer]
Yes, your assumption is correct, the CDN (responsible for updates) will get the new update as soon as we are done with testing and fixing things (there may be some bugs we catch later) and only when it is ready for general availability will that file be updated (this can sometimes be days after the release on GitHub for stability reasons). Soon v4.0.0 will be released, which will change things up a bit.
